### PR TITLE
Move Activate/Deactivate & Data Mapper Extension Variables

### PR DESCRIPTION
**Pull request by Elaina-Lee**
- **Please check if the PR fulfills these requirements**

* [X] The commit message follows our guidelines

- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
moves activate/deactivate as well as data mapper extension variables to designer main
- **What is the current behavior?** (You can also link to an open issue here)
n/a
- **What is the new behavior (if this is a feature change)?**
n/a
- **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

- **Please Include Screenshots or Videos of the intended change**:

### CODE REVIEW

File: apps/vs-code-designer/src/app/commands/dataMapper/DataMapperExt.ts [hunk 1]
```diff
@@ -52,20 +40,20 @@ export default class DataMapperExt {
...
+    ext.dataMapPanelManagers[dataMapName].mapDefinitionData = mapDefinitionData;
 
     // From here, VSIX will handle any other initial-load-time events once receive webviewLoaded msg
   }
 
   public static log(text: string) {
-    DataMapperExt.outputChannel.appendLine(text);
-    DataMapperExt.outputChannel.show();
+    ext.outputChannel.appendLine(text);
+    ext.outputChannel.show();
```
Comment (ccastrotrejo) [inline apps/vs-code-designer/src/app/commands/dataMapper/DataMapperExt.ts:56]: NIT: I have notice that the output channel opens everytime the extension gets activated. Its not a blocker but just wanted to point out if we still want to show it

File: apps/vs-code-designer/src/main.ts [hunk 2]
```diff
@@ -29,6 +31,14 @@ const perfStats = {
 };
 
 export async function activate(context: vscode.ExtensionContext) {
+  //TODO: double check below commands are necessary; poured over from data mapper extension
+  vscode.commands.executeCommand('setContext', 'azureDataMapper.supportedDataMapDefinitionFileExts', supportedDataMapDefinitionFileExts);
```
Comment (ccastrotrejo) [inline apps/vs-code-designer/src/main.ts:35]: NIT: Problably the `azureDataMapper` its going to change `azureLogicAppsStandard`. Also lets store those commands strings in the constants file

**Review by ccastrotrejo — APPROVED** [review]

File: apps/vs-code-designer/src/app/commands/dataMapper/DataMapperExt.ts [hunk 3]
```diff
@@ -52,20 +40,20 @@ export default class DataMapperExt {
       }
     );
 
-    DataMapperExt.panelManagers[dataMapName] = new DataMapperPanel(panel, dataMapName);
-    DataMapperExt.panelManagers[dataMapName].panel.iconPath = {
-      light: Uri.file(path.join(DataMapperExt.context.extensionPath, 'assets', 'wand-light.png')),
-      dark: Uri.file(path.join(DataMapperExt.context.extensionPath, 'assets', 'wand-dark.png')),
+    ext.dataMapPanelManagers[dataMapName] = new DataMapperPanel(panel, dataMapName);
+    ext.dataMapPanelManagers[dataMapName].panel.iconPath = {
+      light: Uri.file(path.join(ext.context.extensionPath, 'assets', 'wand-light.png')),
```
Comment (ccastrotrejo) [inline apps/vs-code-designer/src/app/commands/dataMapper/DataMapperExt.ts:45]: Also just noticed that you might need to move the png's assets aswell

File: apps/vs-code-designer/src/app/commands/dataMapper/DataMapperExt.ts (hunk 1 above)
Comment (Elaina-Lee) [reply apps/vs-code-designer/src/app/commands/dataMapper/DataMapperExt.ts:56]: Would you say we should not show the outputChannel when we are logging?

I noticed in designer, we tend to use `ext.outputChannel.appendLog(message)` ineatead. Should I remove this log function as transition towards that? Not exactly sure if we should keep those `log`, `showWarning` and `showError` functions.

**Review by ccastrotrejo — APPROVED** [review]

**Pull request merged**